extensions/wikia/Search/classes/ResultSet/Base.php

```diff
@@ -3,135 +3,158 @@
  * Class definition for \Wikia\Search\ResultSet\Base
  */
 namespace Wikia\Search\ResultSet;
+
 use \Wikia\Search\Result, \ArrayIterator, \Wikia\Search\MediaWikiService, \Wikia\Search\Utilities;
 use \WikiaException, \Solarium_Result_Select, \WikiaSearchConfig;
+
 /**
  * This is the default class definition -- represents a flat grouping of results, e.g. on-wiki search.
+ *
  * @author relwell
  * @package Search
  * @subpackage ResultSet
  */
-class Base extends EmptySet
-{
+class Base extends EmptySet {
 	/**
 	 * The object that Solarium builds after getting a Solr response
+	 *
 	 * @var Solarium_Result_Select
 	 */
 	protected $searchResultObject;
-	
+
 	/**
 	 * MW service.
+	 *
 	 * @var MediaWikiService
 	 */
 	protected $service;
 
 	/**
 	 * Dependencies are injected here by factory for result, config, and interface, via container.
+	 *
 	 * @param DependencyContainer $container
 	 */
 	protected function configure( DependencyContainer $container ) {
-		$this->searchResultObject  = $container->getResult();
-		$this->searchConfig        = $container->getConfig();
-		$this->service             = $container->getService();
-		$this->results             = new ArrayIterator( array() );
-		$this->resultsFound        = $this->searchResultObject->getNumFound();
-		$this->handleMatchPrepends()
-		     ->setResults( $this->searchResultObject->getDocuments() );
+		$this->searchResultObject = $container->getResult();
+		$this->searchConfig = $container->getConfig();
+		$this->service = $container->getService();
+		$this->results = new ArrayIterator( [] );
+		$this->resultsFound = $this->searchResultObject->getNumFound();
+		$this->handleMatchPrepends()->setResults( $this->searchResultObject->getDocuments() );
 	}
-	
+
 	/**
 	 * set result documents
+	 *
 	 * @param  array $results list of Wikia\Search\Result or Wikia\Search\ResultSet objects
+	 *
 	 * @return Base provides fluent interface
 	 */
 	public function setResults( array $results ) {
-		foreach( $results as $result ) {
+		foreach ( $results as $result ) {
 			$this->addResult( $result );
 		}
+
 		return $this;
 	}
-	
+
 	/**
 	 * Does a little prep on a result object, applies highlighting if exists, and adds to result array.
+	 *
 	 * @param  Result $result
+	 *
 	 * @throws WikiaException
 	 * @return Base provides fluent interface
 	 */
 	protected function addResult( Result $result ) {
 		$id = $result['id'];
 		$highlighting = $this->searchResultObject->getHighlighting();
-		if (        ( $highlighting !== null )
-				&&  ( $hlResult      =  $highlighting->getResult( $id ) )
-				&&  ( $field         =  $hlResult->getField( Utilities::field( 'html' ) ) ) ) {
+		if ( ( $highlighting !== null ) &&
+			( $hlResult = $highlighting->getResult( $id ) ) &&
+			( $field = $hlResult->getField( Utilities::field( 'html' ) ) )
+		) {
 			$result->setText( $field[0] );
 		}
 		if ( $result['created'] ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "created" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)